providers/flagd/src/main/java/dev/openfeature/contrib/providers/flagd/Config.java

```diff
@@ -55,6 +58,22 @@ static int fallBackToEnvOrDefault(String key, int defaultValue) {
         }
     }
 
+    static Evaluator fromValueProvider(Function<String, String> provider) {
+        final String resolverVar = provider.apply(RESOLVER_ENV_VAR);
+        if (resolverVar == null) {
+            return DEFAULT_RESOLVER_TYPE;
+        }
+
+        switch (resolverVar) {
+            case "in-process":
```

Comment:
In .NET I made this case-insensitive. I could go either way since it's not specified. Something to consider.

Reply:
sure, I updated to compare in lowercase - https://github.com/open-feature/java-sdk-contrib/pull/792/commits/f70e6c24d5f650cf4be88e7b011c413cd101c5bd